src/features/NewOrder/NewOrder.js

```diff
@@ -0,0 +1,300 @@
+//WORK ITEMS
+
+// Drop down suggestions for stock symbol input.
+// keep input value (watched) in state?
+// show display if match
+
+// imports
+import React, { useState, useRef } from 'react';
+import { useForm } from 'react-hook-form';
+
+// Styles
+import styles from './NewOrder.module.css';
+
+// sample data
+const realMemeStocks = ['GMSK', 'HYPE', 'MEME', 'MDPC', 'OTHR'];
+const ownedStocks = [
+  {
+    stockName: 'GMSK',
+    stockShares: 7,
+  },
+  {
+    stockName: 'HYPE',
+    stockShares: 23,
+  },
+  {
+    stockName: 'MEME',
+    stockShares: 42,
+  },
+];
+
+const totalCash = 5000.5;
+
+export default function NewOrder() {
+  // use state for showing buy or sell info
+
+  const [showBuyForm, setShowBuyForm] = useState('true');
+
+  return (
+    <div className={styles.formContainer}>
+      <h1>New Order</h1>
+      <div>
+        <button
+          className={styles.buybtn}
+          disabled={showBuyForm}
+          onClick={() => setShowBuyForm(true)}
+        >
+          Buy
+        </button>
+
+        <button
+          className={styles.sellbtn}
+          disabled={!showBuyForm}
+          onClick={() => setShowBuyForm(false)}
+        >
+          Sell
+        </button>
+      </div>
+
+      {showBuyForm ? <BuyForm /> : <SellForm />}
+    </div>
+  );
+}
+
+function BuyForm() {
+  // const [showSearchSuggestions, setSearchSuggestions] = useState("true");
+
+  const { register, handleSubmit, watch, reset, errors } = useForm({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+  });
+
+  const sharePrice = useRef({});
+  sharePrice.current = watch('price', '0.00');
+
+  const quantity = useRef({});
+  quantity.current = watch('quantity', '');
+
+  let totalPrice = sharePrice.current * quantity.current;
+
+  let cashRemaining = totalCash - totalPrice;
+
+  const onSubmit = (data) => {
+    console.log(data);
+    reset();
+  };
+  console.log(errors);
+
+  function findMatch(value) {
+    for (var i = 0; i < realMemeStocks.length; i++) {
+      if (realMemeStocks[i] === value) {
+        i = realMemeStocks.length;
+        return value;
+      }
+    }
+  }
+
+  return (
+    <form className={styles.subFormContainer} onSubmit={handleSubmit(onSubmit)}>
+      <div>
+        <input name="type" ref={register} defaultValue="buy" hidden />
+
+        {errors.symbol && errors.symbol.type === 'validate' && (
+          <div className={styles.error}>Must match a 'real' memestock</div>
+        )}
+        {errors.symbol && errors.symbol.type === 'required' && (
+          <div className={styles.error}>Ticker symbol must be entered</div>
+        )}
+        <label htmlFor="symbol">Stock Ticker Symbol</label>
+
+        <input
+          className={errors.symbol ? styles.inputError : styles.input}
+          name="symbol"
+          type="text"
+          ref={register({
+            required: true,
+            validate: (value) => value === findMatch(value),
```

Comment:
Undoubtedly this works so that's the most important thing. Because this is a simple validator though you may consider doing something like this:
`(value) => realMemeStocks.includes(value)`
or even:
`(value) => realMemeStocks.includes(value.toUpperCase())`